domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/Revurdering.kt

```diff
@@ -356,6 +364,7 @@ sealed class BeregnetRevurdering : Revurdering() {
         override val grunnlagsdata: Grunnlagsdata,
         override val vilkårsvurderinger: Vilkårsvurderinger,
         override val informasjonSomRevurderes: InformasjonSomRevurderes,
+        override val attesteringer: Attesteringshistorikk = Attesteringshistorikk.empty(),
```

Comment:
Er det safere hvis vi begrenser det til at det kun er opprettet som setter attesteringer til default-verdi? Det eneste tilfellet hvor det er relevant for alt mellom opprettet og iverksatt er dersom det eksisterer noe der fra før, i så måte burde vi kanskje sikte på å bruke det som finnes i basen?